src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,20 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    private HibernateUtil() {
+
+    }
+
+    private static SessionFactory initSessionFactory() {
+        return new Configuration().configure().buildSessionFactory();
```

Comment:
It is recommended to use a try-catch block when building the SessionFactory to handle HibernateException and wrap it in a custom unchecked exception.